ext/strings_test.go

```diff
@@ -1079,3 +1101,217 @@ func mustParseDuration(s string) time.Duration {
 		return d
 	}
 }
+
+func unquote(s string) (string, error) {
+	r := []rune(sanitize(s))
+	if r[0] != '"' || r[len(r)-1] != '"' {
+		return "", fmt.Errorf("expected given string to be enclosed in double quotes: %q", r)
+	}
+	var unquotedStrBuilder strings.Builder
+	noQuotes := r[1 : len(r)-1]
+	for i := 0; i < len(noQuotes); {
+		c := noQuotes[i]
+		hasNext := i < len(noQuotes)
```

Comment:
This is always 'false'.

This is a good corner cases where you can add the failing test case, then fix it.

Reply:
Gotcha - I've added a test and a fix.